flink-core/src/main/java/org/apache/flink/util/TemporaryClassLoaderContext.java

```diff
@@ -43,7 +45,8 @@
  * }
  * </pre>
  */
-public final class AutoContextClassLoader implements AutoCloseable {
+@Public
```

Comment:
Why `@Public`? Do we actually expect users to use this?

Reply:
Not sure. I thought it was a simple enough utility such that it would not hurt to expose it, but we can also conservatively start with `@Internal`.